.github/workflows/run-ci.yml

```diff
@@ -15,10 +15,8 @@ jobs:
         ruby: ["2.5", "2.6", "2.7", "3.0"]
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Ruby
-      uses: ruby/setup-ruby@v1
+    - uses: ruby/setup-ruby@v1
       with:
         ruby-version: ${{ matrix.ruby }}
-        bundler-cache: true # runs 'bundle install' and caches installed gems automatically
-    - name: Run rspecs
-      run: bundle exec rspec
+        bundler-cache: true
+    - run: bundle exec rspec
```

Comment:
I think each `run` is explicit enough, so I just deleted `name` as well as the comment for `bundler-cache`.